x-pack/plugins/security_solution/server/lib/detection_engine/rule_schema/model/rule_schemas.mock.ts

```diff
@@ -7,7 +7,7 @@
 
 import { getThreatMock } from '../../../../../common/detection_engine/schemas/types/threat.mock';
 import { getListArrayMock } from '../../../../../common/detection_engine/schemas/types/lists.mock';
-import { getThreatMappingMock } from '../../signals/threat_mapping/build_threat_mapping_filter.mock';
+import { getThreatMappingMock } from '../../rule_types/indicator_match/threat_mapping/build_threat_mapping_filter.mock';
```

Comment:
Just a comment for further refactoring: I believe `rule_types` should depend on `rule_schema`, but not vice versa. Something worth addressing in the future.